client/lib/cart-values/cart-items.js

```diff
@@ -714,6 +724,16 @@ function isRenewal( cartItem ) {
 }
 
 /**
+ * Determines whether a cart item supports privacy
+ *
+ * @param {Object} cartItem - `CartItemValue` object
+ * @returns {boolean} true if item supports privacy
+ */
+function privacyAvailable( cartItem ) {
+	return cartItem.extra && cartItem.extra.privacy_available;
```

Comment:
I like lodash's `get` for stuff like this - you can easily add a default value too: `return get( cartItem, 'extra.privacy_available', true )`. Yes, that's `true` by default - if it's not declared for some reason, then that means it's an old shopping cart from before this PR and backend changes (though, a trip to the backend should update it). Shopping carts never expire, so it's better to provide some sane defaults ;)